src/org/sasylf/Activator.java

```diff
@@ -76,11 +76,11 @@ public static Activator getDefault() {
 	 * @author VonC @ stackoverflow
 	 */
 	public Image getImage(String path) {
-		Image image = Activator.getDefault().getImageRegistry().get(path);

```

Comment:
I don't think this file should be changed in this PR.

Reply:
That's right.  DOn't change it.